test.java

```diff
@@ -0,0 +1,19 @@
+package hackaton.demo;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.concurrent.ThreadLocalRandom;
+import org.junit.jupiter.api.Test;
+
+public class HackatonTest {
+
+  @Test
+  public void testOf() {
```

Comment:
RSPEC-9999: Test methods should be with format of :     nameOfMethod_stateUnderTest_expectedBehavior
<!-- lookout footnote separator -->
_Comment made by the analyzer sonarcheck_analyzer._